app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    cmd, src_file, dst_file = command.split(" ")
+
+    if cmd != "cp" or src_file == dst_file:
```

Comment:
You should check if the source file exists before trying to open it. You can use the `os.path.exists` function for this. If the file does not exist, you should return from the function or raise an error.